src/main/java/com/epam/emotionalhelp/service/impl/QuizResultServiceImpl.java

```diff
@@ -34,6 +34,7 @@
  */
 @Service
 @RequiredArgsConstructor
+@Deprecated
```

Comment:
```suggestion
 *
 * @see QuizResultServiceStreamApproach
 * @deprecated to be removed in future releases
 */
@Service
@RequiredArgsConstructor
@Deprecated(since = "1.0", forRemoval = true)
```